cdap-ui/app/features/dashboard/controllers/dashboard-view-ctrl.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * DashboardCtrl
+ */
+
+angular.module(PKG.name+'.feature.dashboard').controller('DashboardViewCtrl',
```

Comment:
Shouldn't this be UserDashboardCtrl since its being used only for user dashboards?